chain/client/src/sync/state.rs

```diff
@@ -147,6 +147,9 @@ pub struct StateSync {
     /// Maps shard_id to result of applying downloaded state.
     state_parts_apply_results: HashMap<ShardId, Result<(), near_chain_primitives::error::Error>>,
 
+    /// Maps shard_id to result of loading in-memory trie.
+    load_memtrie_results: HashMap<ShardId, Result<(), near_chain_primitives::error::Error>>,
```

Comment:
Is it possible for us to make this ShardUId? We should try using ShardUId where ever possible.

Reply:
I would have to change a lot of existing code to get rid of that, but at least I tried to remove it from the code I added: 08beb0fb18be74567300a3407d14cb3684bdc0db